util/expensivequery/expensivequery.go

```diff
@@ -57,6 +58,7 @@ func (eqh *Handle) Run() {
 	defer ticker.Stop()
 	sm := eqh.sm.Load().(util.SessionManager)
 	record := &memoryUsageAlarm{}
+	serverMemoryQuota := &serverMemoryQuota{}
```

Comment:
Do we need to use an individual goroutine to monitor the global memory status? The expensive_query worker handles too many things